src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss

```diff
@@ -5,7 +5,7 @@ $title-font: 'Montserrat';
 $main-font: 'Open Sans';
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/81276732-190e9b00-905c-11ea-8427-65a291732674.png)
Breakpoint 384

Reply:
![image](https://user-images.githubusercontent.com/54113846/81276840-39d6f080-905c-11ea-98f1-6bf11c36f289.png)
Breakpoint 323

Reply:
![image](https://user-images.githubusercontent.com/54113846/81276901-4fe4b100-905c-11ea-9732-6af08fb89f11.png)
Breakpoint 320

Reply:
Please, fix it according to the mockup.

Reply:
Fixed